src/RRTStar/obstacles.h

```diff
@@ -0,0 +1,24 @@
+#ifndef OBSTACLES_H
+#define OBSTACLES_H
+
+#include "src/constants.h"
```

Comment:
Nit: Please respect the import order: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
Done!